iXBRLViewerPlugin/viewer/src/less/viewer.less

```diff
@@ -6,14 +6,14 @@
 @import (reference) "icons.less";
 
 .ixbrl-highlight:not(.ixbrl-no-highlight) {
-  background-color: @highlight-default !important;
+  background-color: @highlight-default;
```

Comment:
Unfortunately this doesn't work for values within tables. The alternate table row highlighting is applied with inline styles and takes a higher precedence.